config/migrations/2024/20241129081026-fix-incorrect-names.sparql

```diff
@@ -0,0 +1,128 @@
+PREFIX person: <http://www.w3.org/ns/person#>
+PREFIX persoon: <http://data.vlaanderen.be/ns/persoon#>
+PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+
+DELETE {
+  GRAPH ?g {
+    ?persoon persoon:gebruikteVoornaam "Leen" .
+  }
+}
+INSERT {
+  GRAPH ?g {
+    ?persoon persoon:gebruikteVoornaam "Marleen" .
+  }
+}
+WHERE {
+  GRAPH ?g {
+    ?persoon a person:Person ;
+      foaf:familyName "Pollé" ;
+      persoon:gebruikteVoornaam "Leen" .
```

Comment:
For all where items: we do no need to match on these string values, the person URI is good enough right?

Reply:
we don't but I find this is clearer, and it does the same so ...